jenkins/end_to_end_tests

```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
reviewer note: this is NOT the script file that we use, but instead it is a file with no extension that got checked in accidentally.